README.md

```diff
@@ -1,63 +1,73 @@
+tracks our current work and should be compatible with both [Origin master
+branch](https://github.com/openshift/origin/tree/master) and the [most recent
+Origin stable release](https://github.com/openshift/origin/releases). Currently
+that's v1.4 and v1.3.x. In addition to the master branch, we maintain stable
+branches corresponding to upstream Origin releases, e.g.:
+[release-1.2](https://github.com/openshift/openshift-ansible/tree/release-1.2).
+The most recent branch will often receive minor feature backports and fixes.
+Older branches will receive only critical fixes.
 
 Releases are tagged periodically from active branches and are versioned 3.x
 corresponding to Origin releases 1.x. We unfortunately started with 3.0 and it's
 not practical to start over at 1.0.
 
-##Setup
-- Install base dependencies:
-  - Requirements:
-    - Ansible >= 2.1.0 though 2.2 is preferred for performance reasons.
+## Directory structure
+
+- [bin/cluster](https://github.com/openshift/openshift-ansible/tree/master/bin/cluster) - Python script to easily create clusters
+- [docs/](https://github.com/openshift/openshift-ansible/tree/master/docs) - documentation for the project
+- [filter_plugins/](https://github.com/openshift/openshift-ansible/tree/master/filter_plugins) - custom filters used to manipulate data in Ansible
+- [inventory/](https://github.com/openshift/openshift-ansible/tree/master/inventory) - Ansible dynamic inventory scripts
+- [playbooks/](https://github.com/openshift/openshift-ansible/tree/master/playbooks) - host-type Ansible playbooks (launch, config, destroy, vars)
+- [roles/](https://github.com/openshift/openshift-ansible/tree/master/roles) - shareable Ansible tasks
+
+For more information, see the [contribution guide](CONTRIBUTING.md).
+
+## Setup
+
+1. Install base dependencies:
+
+    ***
+    Requirements:
+    - Ansible >= 2.1.0 (>= 2.2 is preferred for performance reasons)
     - Jinja >= 2.7
+    ***
+
+    Fedora:
+    ```
+    dnf install -y ansible pyOpenSSL python-cryptography
+    ```
+
+    macOS:
+    ```
+    brew install ansible
+    ```
+
+2. Setup for a specific cloud:
 
-  - Fedora:
-  ```
-    dnf install -y ansible-2.1.0.0 pyOpenSSL python-cryptography
-  ```
-
-  - OSX:
-  ```
-    # Install ansible 2.1.0.0 and python 2
-    brew install ansible python
-  ```
-- Setup for a specific cloud:
   - [AWS](http://github.com/openshift/openshift-ansible/blob/master/README_AWS.md)
   - [GCE](http://github.com/openshift/openshift-ansible/blob/master/README_GCE.md)
   - [local VMs](http://github.com/openshift/openshift-ansible/blob/master/README_libvirt.md)
+  - Bring your own host deployments:
+      - [OpenShift Enterprise](https://docs.openshift.com/enterprise/latest/install_config/install/advanced_install.html)
+      - [OpenShift Origin](https://docs.openshift.org/latest/install_config/install/advanced_install.html)
+      - [Atomic Enterprise](http://github.com/openshift/openshift-ansible/blob/master/README_AEP.md)
```

Comment:
> and the linked README, if you like.

Ah, you've said so in the begging alright, I'll get rid of that as well to lower confusion.